lib/frame-builder.js

```diff
@@ -182,7 +182,8 @@ frame_builder[C.FRAME_TYPE.READ_PARAMETER] = function(frame, builder) {
   }
 };
 
-frame_builder[C.FRAME_TYPE.VERSION] = function(_frame, _builder) {
+frame_builder[C.FRAME_TYPE.VERSION] = function(_frame, builder) {
+  builder.appendUint32LE(0);
```

Comment:
```suggestion
  builder.appendUInt32LE(0);
```
Tested this and got an error, that the function does not exist